nydusify: support --push-chunk-size option

imeoer opened the pull request:
Reference: https://github.com/containerd/containerd/pull/9405

Will replace containerd dep to upstream version if the PR can be merged.

## Relevant Issue (if applicable)
_If there are Issues related to this PullRequest, please list it._

## Details
_Please describe the details of PullRequest._

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

Desiki-high commented:
Maybe we can add `go.work.sum` into `.gitignore` since it always changes too large. We only need it to work well locally.

Desiki-high commented on go.work.sum:
Maybe we can add go.work.sum into .gitignore since it always changes too large. We only need it to work well locally.

imeoer committed:
nydusify: support --push-chunk-size option

Reference: https://github.com/containerd/containerd/pull/9405

Will replace containerd dep to upstream version if the PR can be merged.

Signed-off-by: Yan Song <yansong.ys@antgroup.com>

imeoer committed:
contrib: fix golangci lint for ctr-remote

Fix the lint check error by updating containerd package:

```
golangci-lint run
Error: commands/rpull.go:89:2: SA1019: log.G is deprecated: use [log.G]. (staticcheck)
	log.G(pCtx).WithField("image", ref).Debug("fetching")
	^
```

Signed-off-by: Yan Song <yansong.ys@antgroup.com>

adam3q reviewed (approved):
Thanks!
